website/src/components/AddDevice.tsx

```diff
@@ -94,16 +99,27 @@ export const AddDevice = observer(class AddDevice extends React.Component<Props>
           <CardHeader title="Add A Device" />
           <CardContent>
             <form onSubmit={this.submit}>
-              <FormControl error={!!this.error} fullWidth>
+              <FormControl fullWidth>
                 <InputLabel htmlFor="device-name">Device Name</InputLabel>
                 <Input
                   id="device-name"
                   value={this.deviceName}
                   onChange={(event) => (this.deviceName = event.currentTarget.value)}
                   aria-describedby="device-name-text"
                 />
-                <FormHelperText id="device-name-text">{this.error}</FormHelperText>
+                <FormHelperText id="device-name-text">Any name to your liking</FormHelperText>
+              </FormControl>
+              <FormControl fullWidth>
+                <InputLabel htmlFor="device-publickey">Device Public Key</InputLabel>
+                <Input
+                  id="device-publickey"
+                  value={this.devicePublickey}
+                  onChange={(event) => (this.devicePublickey = event.currentTarget.value)}
+                  aria-describedby="device-publickey-text"
+                />
+                <FormHelperText id="device-publickey-text">You may extract your public key from your given private key with: wg pubkey &lt; private.key</FormHelperText>
```

Comment:
I think this help message is a bit too advanced for the average user. Those who were able to generate a privkey with the `wg` tool and output it in a file likely already know how to get a pubkey from it.

Would it be safe to assume that people who would make use of this functionality, either because they are tech-savvy enough to know about public and private keys, or because they got instructions from their admins, already know how and where to get their keys?
In this case the comment could be something like `Put your public key to a pre-generated private key here. Replace the private key in the config after downloading it.`